issue_05/script.js

```diff
@@ -9,3 +9,22 @@ const contents = [{
     alt: "画像2",
     text: "メッセージ"
 }];
+
+const ul = document.getElementById('js-ul');
+const promise = new Promise((resolve) => {
```

Comment:
IMO
It's better to make the function name a little more specific.

https://qiita.com/KeithYokoma/items/2193cf79ba76563e3db6

Reply:
Thanks for the reference article too!
It was a bit long, but I've revised it.
https://github.com/yoshikawa-akr/js-lessons/pull/5/commits/4201b64e7bff37e7d656db8582c600921220fe2f

Reply:
It's a little long, so how about `createList`?